stylus/settings/palette.styl

```diff
@@ -199,6 +199,7 @@ html, .CozyTheme--normal
     Styleguide Settings.theme.components
     */
     --dividerColor: #1D212A1F // grey[900] with 12% opacity
+    --divider: rgba(29, 33, 42, 0.12)
```

Comment:
What is the difference between divider and dividerColor ?

Reply:
from figma, it seems that dividerColor is the right one? 
![Uploading Capture d’écran 2021-09-14 à 08.42.21.png…]()